src/service/jobRoleService.ts

```diff
@@ -2,15 +2,15 @@ import axios from "axios";
 import type { JobRole } from "../model/jobRole";
 
 export const jobRoleService = {
-    async getJobRoles(): Promise<JobRole[]> {
+    async getJobRoles(token?: string): Promise<JobRole[]> {
```

Comment:
I did that so it would compile since the token in the session could be undefined. Is there a better approach rather than just checking in every single controller whether the session exists, given each authorised route already has middleware to ensure it does?